pages/api/school/create.js

```diff
@@ -0,0 +1,31 @@
+import prisma from '@prisma/index'; // Prisma client
+import { getSession } from 'next-auth/client'; // Session handling
+
+export default async (req, res) => {
+  // Collect session from request
+  const session = await getSession({ req });
+
+  // If user is authenticated and is an admin
+  if (session && session.isAdmin) {
+    // Collect name of school
+    const { name: name } = req.body;
```

Comment:
No need to rename when destructuring object.
```suggestion
    const { name } = req.body;
```